changelogs/fragments/254_variable_warnings.yml

```
@@ -0,0 +1,5 @@
+---
+minor_changes:
+- firewalld - Show warning message that variable type of ``masquerade`` and
+  ``icmp_block_inversion`` will be changed from ``str`` to ``boolean``
+  in the future release.
```

```suggestion
  in the future release (https://github.com/ansible-collections/ansible.posix/pull/254).
```
it'd be easier for users to find the context